mu/modes/circuitpython.py

```diff
@@ -72,6 +72,7 @@ class CircuitPythonMode(MicroPythonMode):
         (0x239A, 0x80AC, None, "Unexpected Maker FeatherS2"),
         (0x303A, 0x8002, None, "Unexpected Maker TinyS2"),
         (0x054C, 0x0BC2, None, "Spresense"),
+        (0x16d0, 0x08c7, None, "Tiny 2040")
```

Comment:
I believe the tests are failing because the black formatter wants a comma here.
You can run the test locally with `make check` (although if you are using Python 3.5 black won't run as it is not available)